CHANGELOG.md

```diff
@@ -12,6 +12,8 @@ For information on changes in released versions of Teku, see the [releases page]
 
 ### Breaking Changes
 
+- Network configs updated following Consensus Specs changes. If you run custom network, you will need to add [lines with network parameters](https://github.com/ConsenSys/teku/blob/475986c523b606c6936d7b4207c1da920ad82ea0/ethereum/spec/src/main/resources/tech/pegasys/teku/spec/config/configs/mainnet.yaml#L98-L125) to your custom config. If you are using a remote validator `auto` network feature, you will need to update both Beacon Node and Validator Client.  
```

Comment:
I like this, we probably just need to update the URL depending on how many spec entries we end up finishing before release...